observations/obs_converters/NCEP/prep_bufr/prep_bufr.rst

```diff
@@ -1,67 +1,315 @@
+For those with access to Derecho looking for a quick overview of all steps in the conversion process see the **Quickstart
+Instructions** below.
+
+Quickstart Instructions
+-----------------------
+
+This section provides a complete list of steps within the conversion process (prepbufr file to obs_seq.out) performed on Derecho. 
+It is only recommended if you have a Derecho account, or prefer a comprehensive list of the required steps/commands. 
+Otherwise please proceed to the **Prepbufr Overview** section for more details.
+
+
+- The following Derecho modules were loaded during the compiling process. This exact setup
+  is not mandatory, but given for reference.
+
+   ::
+
+     Currently Loaded Modules:
+        1) ncarenv/23.09 (S)   3) intel/2023.2.1        5) cray-mpich/8.1.27  7) netcdf/4.9.2   9) ncl/6.6.2 
+        2) craype/2.7.23       4) ncarcompilers/1.0.0   6) hdf5/1.12.2        8) nco/5.2.4 
+
+-  Go into your DART prep_bufr observation converter directory and
+   install the PREPBUFR utilities as follows:
+
+   ::
+
+      cd $DART_DIR/observations/obs_converters/NCEP/prep_bufr
+      ./install.sh
+
+   Confirm the exe directory contains the executables ``prepbufr.x``, ``prepbufr_03z.x``,
+   ``grabbufr.x`` and ``cword.x``.
+
+-  Go to ``$DART_DIR/build_templates/`` directory and use the intel build template
+   for compiling DART
+
+   ::
+
+      cd $DART_DIR/build_templates
+      cp mkmf.template.intel.linux mkmf.template
+
+-  Go to the ``$DART_DIR/observations/obs_converters/NCEP/prep_bufr/work/``
+   directory and run *quickbuild.sh* to build the DART
+   PREPBUFR-to-intermediate-file observation processor:
+
+   ::
+
+      cd $DART_DIR/observations/obs_converters/NCEP/prep_bufr/work
+      ./quickbuild.sh
+
+
+-  Obtain the PREPBUFR observations for your desired time. You can go to the
+   the campaign collections directory as:
+
+   ::
+
+      cd /glade/campaign/collections/rda/data/d090000/2017/
+      cp A25366-201706prepqmB.tar $DART_DIR/observations/obs_converters/NCEP/prep_bufr/data/prepqm
 
-Translating NCEP PREPBUFR files into DART obs_seq.out files (input file to filter) is a 2 stage process. The first stage
-uses NCEP software to translate the PREPBUFR file into an intermediate text file. This is described in this document.
-The second step is to translate the intermediate files into obs_seq.out files, which is done by create_real_obs, as
-described in :doc:`../ascii_to_obs/create_real_obs` .
 
-Instructions
-------------
+   Alternatively, download the files directly from the `NSF NCAR Research Data
+   Archive <NCEP+NCAR_obs_>`_ page for the
+   NCEP/NSF NCAR Global Reanalysis Products. Register on the site, click on
+   the "Data Access" tab, and locate the "prepqm: BUFR observations files".
+   Use a similar approach to  obtain the `RDA ds377 <https://rda.ucar.edu/datasets/d337000/>`__
+   prepbufr files. Locate and download the GDAS PREPBUFR files of your choice.   
+  
```

Comment:
@kdraeder  just added this dataset as a reference in 
https://github.com/NCAR/DART/blob/aee5c059fa4a05f5bb6672457cc6c9e29d470faf/guide/references.rst#L16-L17

Reply:
Thanks for mentioning this.  I just realized I blew off @kdraeder , request for comment on the RDA PR a couple weeks back.   Sorry !! -- this was unintentional, I am trying to juggle too many things right now.  I will restore the original link that Kevin generated.